README.md

```diff
@@ -7,7 +7,45 @@ This repository provides KubeVirt-specific Tekton tasks, which focus on:
 - Executing commands in VMs
 - Manipulating disk images with virt-customize [WIP]
 
-## Usage and Deployment
+## Deployment
+
+### On Kubernetes
+
+Run the following command to install KubeVirt Tekton tasks in an active namespace.
+
+The installation manifest includes required RBAC roles and service accounts.
+The following command needs to be run in every namespace the tasks will be used in, because these service accounts need to be created for each new namespace.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubevirt/kubevirt-tekton-tasks/main/manifests/kubernetes/namespace-rbac/kubevirt-tekton-tasks.yaml
+```
+
+If the pipeline needs to access resources (VMs, PVCs, etc.) across different namespaces (cluster privileges), the following commands need to be run instead.
```

Comment:
I updated the readme a bit. Please let me know if it is clearer.

- the `TARGET_NAMESPACE` is a namespace where pipelineruns/taskruns will be run, but the pipeline can operate on resources (VMs) in different namespaces
- the pipeline is run in `TARGET_NAMESPACE` and as an example it can be run by admin/bot to create VMs for users in the cluster (expecting that each user has its own namespace). So the user will just see a new VM to appear, but will not see the pipeline/pipelinerun.